### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**

### CODE REVIEW

File: test/src/test/java/hudson/model/MyViewsPropertyTest.java [hunk 1]
```diff
@@ -51,7 +51,7 @@ public void testReadResolve() throws IOException {
         property.setUser(user);
         user.addProperty(property);
         property.readResolve();
-        assertNotNull("Property should contains " + AllView.DEFAULT_VIEW_NAME + " defaultly.", property.getView(AllView.DEFAULT_VIEW_NAME));
+        assertNotNull("Property should contain " + AllView.DEFAULT_VIEW_NAME + " default.", property.getView(AllView.DEFAULT_VIEW_NAME));
```
Comment (daniel-beck) [inline test/src/test/java/hudson/model/MyViewsPropertyTest.java]: … by default.

File: war/src/main/webapp/css/style.css [hunk 2]
```diff
@@ -650,7 +650,7 @@ div.behavior-loading {
   border-radius: 100%;
   -webkit-animation: sk-scaleout 1.0s infinite ease-in-out;
   animation: sk-scaleout 1.0s infinite ease-in-out;
-  box-shaodw:#fff 0 0 0 10px
+  box-shadow:#fff 0 0 0 10px
```
Comment (daniel-beck) [inline war/src/main/webapp/css/style.css]: Will change appearance. What's the effect of this?

File: core/src/main/java/hudson/org/apache/tools/tar/TarInputStream.java [hunk 3]
```diff
@@ -246,7 +246,7 @@ public TarEntry getNextEntry() throws IOException {
             this.currEntry = new TarEntry(headerBuf);
 
             if (this.debug) {
-                System.err.println("TarInputStream: SET CURRENTRY '"
+                System.err.println("TarInputStream: SET currENTRY '"
```
Comment (daniel-beck) [inline core/src/main/java/hudson/org/apache/tools/tar/TarInputStream.java]: Why?

**Comment by daniel-beck** [comment]
Requesting reviews by @jglick (merges of security fixes) and @olivergondza (LTS backports).

**Comment by oleg-nenashev** [comment]
sigh, sorry for so many typos in the changelog

File: war/src/main/webapp/css/style.css (hunk 2 above)
Comment (jsoref) [inline war/src/main/webapp/css/style.css]: (I obviously don't know -- but the current line is just bloat)

File: test/src/test/java/hudson/model/MyViewsPropertyTest.java (hunk 1 above)
Comment (jsoref) [inline test/src/test/java/hudson/model/MyViewsPropertyTest.java]: Sure

File: core/src/main/java/hudson/org/apache/tools/tar/TarInputStream.java (hunk 3 above)
Comment (jsoref) [inline core/src/main/java/hudson/org/apache/tools/tar/TarInputStream.java]: To match: https://github.com/jenkinsci/jenkins/blob/08def67a18eee51de9f3f99bc2a792fee1c160e0/core/src/main/java/hudson/org/apache/tools/tar/TarInputStream.java#L215
```java
                System.err.println("TarInputStream: SKIP currENTRY '"
```

**Comment by jsoref** [comment]
@oleg-nenashev: speaking of which, one of the changelogs includes a comment about fixing a misspelling, sadly it specifies how it was misspelled.

As a suggestion going forward (and possibly for that one for now): please don't tell people how you misspelled things -- it just makes tools like mine annoyed. All anyone cares about (if they care at all) is the correct spelling.

I omitted that spelling correction, because, as a human, I don't offer to correct things that are intentionally misspelled, and that one counted...

File: core/src/main/java/hudson/tasks/UserNameResolver.java [hunk 4]
```diff
@@ -58,7 +58,7 @@
      *
      * <p>
      * When multiple resolvers are installed, they are consulted in order and
-     * the search will be over when a name is found by someoene.
+     * the search will be over when a name is found by someene.
```
Comment (jsoref) [inline core/src/main/java/hudson/tasks/UserNameResolver.java]: Sorry, this is wrong (the fixes are all manual, so clearly my cursor was off by one when I suggested the correction).

File: core/src/main/java/hudson/org/apache/tools/tar/TarInputStream.java (hunk 3 above)
Comment (daniel-beck) [inline core/src/main/java/hudson/org/apache/tools/tar/TarInputStream.java]: :+1:

**Comment by orrc** [comment]
Looks ok to me. Though at merge time I'm sure we could live with one squashed commit, rather than 150+ commits… :)

File: war/src/main/webapp/css/style.css (hunk 2 above)
Comment (orrc) [inline war/src/main/webapp/css/style.css]: > (I obviously don't know -- but the current line is just bloat)

It's up to you as submitter to show what impact this has, which is what I guess Daniel was getting at.

Anyway, I tested this change, and it just adds a fairly subtle white halo around the circle animation:
image

**Comment by jsoref** [comment]
re spelling errors in changelogs, sorry, that was exim not jenkins:
https://github.com/Exim/exim/blob/80fea873648ca2ab2e592999a336c59cf054ab55/src/src/eximstats.src#L32
`# 1997-03-10  Fixed typo "destinationss"`

**Comment by daniel-beck** [comment]
I was wondering when we started going into detail like that :laughing:

**Review by olivergondza — APPROVED** [review]

**Comment by daniel-beck** [comment]
@jglick WDYT? Any problems expected with security fix merging due to this?

**Review by oleg-nenashev — APPROVED** [review]
Looks good to me :+1:. Needs to be squashed and integrated to the weekly after the security release.

File: core/src/main/java/hudson/util/ProcessTree.java [hunk 5]
```diff
@@ -1110,7 +1110,7 @@ private void parse() {
                     // for some reason, I was never able to get sysctlbyname work.
 //        if(LIBC.sysctlbyname("kern.argmax", argmaxRef.getPointer(), size, NULL, _)!=0)
                     if(LIBC.sysctl(new int[]{CTL_KERN,KERN_ARGMAX},2, argmaxRef.getPointer(), size, NULL, _)!=0)
-                        throw new IOException("Failed to get kernl.argmax: "+LIBC.strerror(Native.getLastError()));
+                        throw new IOException("Failed to get kernel.argmax: "+LIBC.strerror(Native.getLastError()));
```
Comment (jglick) [inline core/src/main/java/hudson/util/ProcessTree.java]: Actually `kern.argmax` is correct here (see above).

File: test/src/test/java/hudson/model/LabelLoadStatisticsQueueLengthTest.java [hunk 6]
```diff
@@ -188,7 +188,7 @@ public void queueLengthReflectsJobsAssignedLabel() throws Exception {
     private FreeStyleProject createTestProject() throws IOException {
         FreeStyleProject project = j.createFreeStyleProject(PROJECT_NAME);
         // In order to queue multiple builds of the job it needs to be
-        // parameterised.
+        // parameterized.
```
Comment (jglick) [inline test/src/test/java/hudson/model/LabelLoadStatisticsQueueLengthTest.java]: Only if we are standardizing on American spelling I guess.

File: test/src/test/java/hudson/model/WorkspaceCleanupThreadTest.java [hunk 7]
```diff
@@ -149,7 +149,7 @@
         assertFalse(ws.exists());
     }
 
-    @Test @WithoutJenkins public void reocurencePeriodIsInhours() {
+    @Test @WithoutJenkins public void recurrencePeriodIsInhours() {
```
Comment (jglick) [inline test/src/test/java/hudson/model/WorkspaceCleanupThreadTest.java]: `recurrencePeriodIsInHours`

**Review by jglick — APPROVED** [review]

**Comment by daniel-beck** [comment]
@oleg-nenashev Looks like we're ready to merge. Do you want this to be squashed?

**Comment by oleg-nenashev** [comment]
Yep, I do. Merging

**Pull request merged**